martian-filetypes/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "martian-filetypes"
-version = "0.23.0"
+version = "0.25.0"
```

Comment:
I bumped here from `0.23.0` to `0.25.0` to keep in sync with other crates, or is it OK if they are not the same?

Reply:
My feeling on this is, in priority order:
1. Don't bump the version if there's no change to the crate (perhaps obvious)
2. If the change isn't breaking, make sure that's reflected in the crate bump.  That is, if the change is not breaking, then only change the z.
3. It's probably a good thing to skip versions where required in order to keep the version numbers in sync.